src/util.h

```diff
@@ -101,9 +101,7 @@ inline int CalcBitRateBitsPerSecFromCodedBytes ( const int iCeltNumCodedBytes, c
 
 QString GetVersionAndNameStr ( const bool bDisplayInGui = true );
 QString MakeClientNameTitle ( QString win, QString client );
-#ifndef HEADLESS
 QString TruncateString ( QString str, int position );
```

Comment:
Maybe it was the headless client...

Reply:
Yep, it was the headless build as `TruncateString` was previously only available to GUI builds. The part you cited was half of the fix already, but I initially missed the second part, which I've just added.